src/support/HISTORY.md

```diff
@@ -1,6 +1,10 @@
 Release History
 ===============
 
+2.0.0
+-----
+* Upgrade CLI Support extension to new Support RP API 2024-04-01
```

Comment:
Current update log is rather general, please log the subgroup level change into HISTORY notes.
![image](https://github.com/Azure/azure-cli-extensions/assets/8137253/72ca4315-b7eb-4065-a59d-f5ad3631e288)